nwbwidgets/misc.py

```diff
@@ -461,16 +466,16 @@ def update(
                 self.trials,
                 start_label,
                 start_label,
-                before + sigma_in_secs * 4,
-                after + sigma_in_secs * 4,
+                start + sigma_in_secs * 4,
+                end + sigma_in_secs * 4,
                 order,
                 progress_bar=progress_bar,
             )
             show_psth_smoothed(
                 expanded_data,
                 axs[1],
-                before + sigma_in_secs * 4,
-                after + sigma_in_secs * 4,
+                start + sigma_in_secs * 4,
```

Comment:
should this be `start - sigma_in_secs * 4`?

Reply:
exactly, fixed it now, thanks!